src/app/courses/components/add-course/add-course.component.html

```diff
@@ -1,6 +1,38 @@
-<p>
-  Add course
-</p>
+<form class="example-form">
+<div class="container">
+  <div class="header">
+    <span class="title">Add Course</span>
+    <span class="btn-close" (click)="cancel()">X</span>
+  </div>
+  <mat-divider></mat-divider>
 
+  <div class="fields">
+    <mat-form-field class="row title">
+      <input matInput [(ngModel)]="course.title" name="title" placeholder="Title *" autocomplete="off">
+    </mat-form-field>
+    <mat-form-field class="row details">
+      <textarea matInput [(ngModel)]="course.description" name="description" placeholder="Details *" rows="4" autocomplete="off"></textarea>
+    </mat-form-field>
+    <mat-form-field class="date">
+      <input matInput [matDatepicker]="picker" [(ngModel)]="course.dateCreation" name="dateCreation" placeholder="Date *">
+      <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+      <mat-datepicker #picker></mat-datepicker>
+    </mat-form-field>
+    <mat-form-field class="duration">
+        <input matInput [(ngModel)]="course.duration" name="duration" placeholder="Duration *" autocomplete="off" onlyNumbers value="">
+    </mat-form-field>
+    <span>{{course.duration | duration}}</span>
```

Comment:
It's better to hide this field, while `course.duration` is not specified.
Also, it would be better to write "min", not only "sec", to avoid misunderstanding.